### PR TITLE
Sprint 38 TLT-1155 Add people UI

### PR DESCRIPTION
**Pull request by rascalking**
Allow the user to add a person to a course from the course people view.

NOTE: this depends on the task/elliottyates/tlt-1155/add_people branch of icommons-rest-api.

### CODE REVIEW

File: course_info/static/course_info/js/controllers/PeopleController.js [hunk 1]
```diff
@@ -10,18 +11,252 @@
...
+                        })
+                        .error(function(data, status, headers, config, statusText) {
+                            // log it, then display a warning
+                            $scope.handleAjaxError(data, status, headers, config,
+                                                   statusText);
+                            $scope.partialFailures.push({
+                                searchTerm: searchTerm,
+                                text: 'Add to course seemed to succeed, but ' +
+                                      'we received an error trying to retrieve ' +
+                                      "the user's course details.",
```
Comment (eparker71) [inline course_info/static/course_info/js/controllers/PeopleController.js]: minor: switched to double quotes, maybe switch to single to keep it consistent.

Comment (rascalking) [inline course_info/static/course_info/js/controllers/PeopleController.js]: That's to avoid having to escape the apostrophe in the string.

Comment (eparker71) [inline course_info/static/course_info/js/controllers/PeopleController.js]: ah, sure, I didnt' see that :)

**Comment by eparker71** [comment]
:+1:

**Pull request merged**